bugbot/rules/assignee_no_login.py

```diff
@@ -68,6 +78,34 @@ def get_bugs(self, *args, **kwargs):
 
         return bugs
 
+    def fetch_bug_histories(self, bug_ids):
+        bug_histories = {}
+
+        def history_handler(bug):
+            bug_id = str(bug["id"])
+
+            history_entries = []
+            for history in bug["history"]:
+                for change in history["changes"]:
+                    history_entries.append(
+                        {
+                            "field_name": change["field_name"],
+                            "removed": change["removed"],
+                            "added": change["added"],
+                            "when": history["when"],
+                            "who": history["who"],
+                        }
+                    )
+
+            bug_histories[bug_id] = history_entries
+
+        Bugzilla(
+            bugids=bug_ids,
+            historyhandler=history_handler,
+        ).get_data().wait()
+
+        return bug_histories
```

Comment:
Why do we need to retrieve the history in a separate query?

Reply:
I realized I wasn't populating the `data` dictionary with the `history`, but I've changed that here and removed the separate history query: 6b2ea7f3d504fd06944cb56bcd554906d04388da